osidb/models.py

```diff
@@ -1872,6 +1887,16 @@ def _validate_affect_status_resolution(self, **kwargs):
                 f"affectedness/resolution combination: {self.resolution} is not a valid resolution "
                 f"for {self.affectedness}."
             )
+        
+        if (
+            self.resolution == Affect.AffectResolution.DEFER
+            and self.impact != Impact.LOW
+        ):
+            raise ValidationError(
+                f"Affect ({self.uuid}) for {self.ps_module}/{self.ps_component} has resolution "
+                f"{self.resolution} {Affect.AffectResolution.DEFER}, but has {self.impact} impact/severity."
+                f"{Affect.AffectResolution.DEFER} is only valid on affects with {Impact.LOW} impact/severity."
+            )
```

Comment:
This second check seems duplicite to me. Also the it extends the validation scope as it was originally supposed to only check the status and resolution combinations - not impact.

Reply:
Indeed, I had written a separate dedicated validation where I separated this logic but hadn't deleted my original code here when I first created the PR. I have since added a commit to remove it to this PR which I can squash later in the review process.